src/daggy.js

```diff
@@ -46,22 +46,22 @@
     var typeRep = proto.constructor = {
       toString: typeRepToString,
       prototype: proto,
-      is: isType,
       '@@type': typeName
     };
+    typeRep.is = isType(typeName);
```

Comment:
Why not define `typeRep.is` in its old location?

Reply:
Oops, pretty sure that's just because I'd started to implement that in a slightly different manner then didn't move it back.